cli/src/main/java/io/quarkiverse/tekton/cli/GenerateCommand.java

```diff
@@ -23,12 +23,19 @@ public Properties getBuildSystemProperties() {
     @Override
     public void process(List<HasMetadata> resources) {
         String json = Serialization.asJson(resources);
-        String yml = Serialization.asYaml(resources);
+
+        // Iterate through the list of the resources to process them individually to include ---
```

Comment:
Let's move this directly in the `asYaml` implementation so that we ensure that the experience is the same throughout the project.

Reply:
To care that the method `asYaml` is coming from the Fabric8 Kube client class `io.fabric8.kubernetes.client.utils.Serialization.class` !

Do we have then to perform the change first on Fabric8 Kube client, wait till we got a new release to update this PR ? @iocanel

Reply:
We shouldn't be using that `Serialization`. We have a customized version of that in `io.qarkiverse.tekton.common.utils`. We should use that instead.

Reply:
Fixed and added the change to our util class: aa85f18
WDYT ? @io